Tetrahedral_remeshing/include/CGAL/Tetrahedral_remeshing/Uniform_sizing_field.h

```diff
@@ -19,8 +19,11 @@
 
 namespace CGAL
 {
+namespace Tetrahedral_remeshing
+{
 template <class Kernel>
-class Uniform_sizing_field : Sizing_field<Kernel>
+class Uniform_sizing_field
```

Comment:
Why does this class have no doxygen ?  Note that it is listed in the reference manual without link.